components/menu/menu-item-link.js

```diff
@@ -129,19 +136,27 @@ class MenuItemLink extends (newWindowIconEnabled ? LinkMixin(MenuItemMixin(LitEl
 		return null;
 	}
 
+	// remove this function when cleaning up GAUD-8369-menu-item-link-click-changes
 	_onClick() {
 		if (this.shadowRoot) this.shadowRoot.querySelector('a').dispatchEvent(new CustomEvent('click'));
 	}
 
 	_onKeyDown(e) {
-		if (e.keyCode === this.__keyCodes.ENTER || e.keyCode === this.__keyCodes.SPACE) {
-			const target = this._getTarget();
-			if (target === '_parent') {
-				window.parent.location.assign(this.href);
-			} else if (target === '_top') {
-				window.top.location.assign(this.href);
-			} else {
-				window.location.assign(this.href);
+		if (menuItemClickChangesEnabled) {
+			if (e.keyCode === this.__keyCodes.ENTER || e.keyCode === this.__keyCodes.SPACE) {
+				this.shadowRoot.querySelector('a').click();
+			}
+		} else { // remove this block when cleaning up GAUD-8369-menu-item-link-click-changes
+			super._onKeyDown(e);
```

Comment:
This is the change I plan to add to [homepage-widget-expansion-pack](https://github.com/Brightspace/homepage-widget-expansion-pack/blob/92b293a8b58257c899eee619cc0656848a55a381/components/content-navigator/d2l-cn-menu-item-link.js#L105) as well.